hedyorg/hedy
# [UI] Add a New Year's adventure to Level 11

Description
Adds a New Year's story to Level 11 Quiz

Fixes issue https://github.com/hedyorg/hedy/issues/3455

How to test
Go to http://localhost:8080/hedy/11#years
See that the New Year's story appears and that editing the code and then clicking the "run_code_button" allows for output to be displayed 

Checklist
[X] Contains one of the PR categories in the name
[X] Describes changes in the format above
[X] Links to an existing issue or discussion
[X] Has a "How to test" section
